src/Plugin/Block/SkipNavAnchorBlock.php

```diff
@@ -24,8 +24,9 @@ public function build() {
     return [
       'anchor' => [
         '#title' => $this->t('Main Content'),
-        '#type' => 'link',
-        '#url' => Url::fromUri('internal:#' . $id),
+        '#type' => 'html_tag',
+        '#tag' => 'div',
+        '#value' => $this->t('Main Content'),
```

Comment:
since you have the `#value`, the `#title` can be removed.

Reply:
oh and the CodeClimate fix & tests

Reply:
Thanks! will fix.